packages/task/src/common/task-protocol.ts

```diff
@@ -15,19 +15,23 @@
  ********************************************************************************/
 
 import { JsonRpcServer } from '@theia/core/lib/common/messaging/proxy-factory';
```

Comment:
this file should contain only definition of remote services exposed over JSON-RPC and supporting types

please move everything else to the frontend or backend as classes